### PR TITLE
Add banner for MLH events

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

File: src/components/EventCard.js [hunk 1]
```diff
@@ -54,18 +50,15 @@ const EventCard = Card.withComponent(Tilt).extend.attrs({
...
-  f: 1,
+  bg: 'white',
+  w: '6em',
 })`
-  position: relative;
-  left: 0;
-  top: 0;
-  border-bottom-right-radius: ${props => props.theme.space[2]}px;
+border-top-right-radius: ${props => props.theme.space[2]}px;
+border-bottom-right-radius: ${props => props.theme.space[2]}px;
```
Comment (lachlanjc) [inline src/components/EventCard.js]: Use radii instead of space :)

(and indent, but nbd)

File: static/mlh-logo-grayscale.svg [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="343px" height="145px" viewBox="0 0 343 145" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 3.8.3 (29802) - http://www.bohemiancoding.com/sketch -->
+    <title>mlh-logo-grayscale</title>
```
Comment (lachlanjc) [inline static/mlh-logo-grayscale.svg]: Run svgo on this file: https://www.npmjs.com/package/svgo

Comment (maxwofford) [reply static/mlh-logo-grayscale.svg]: Wow, that did a ridiculously good job. Thanks for telling me about this 🤩

**Comment by maxwofford** [comment]
Thanks for the review @lachlanjc!

**Pull request merged**